R/weight-streetnet.R

```diff
@@ -1,9 +1,64 @@
 #' Weight a street network according to a specified weighting profile.
 #'
-#' Weight (or re-weight) an \pkg{sf} or `SC` (`silicate`)-formatted OSM street
-#' network according to a named profile, selected from (foot, horse, wheelchair,
-#' bicycle, moped, motorcycle, motorcar, goods, hgv, psv), or a customized
-#' version derived from those.
+#' Weight (or re-weight) an \pkg{sf} or \pkg{silicate} *("SC") formatted OSM
+#' street network according to a named profile, selected from (foot, horse,
+#' wheelchair, bicycle, moped, motorcycle, motorcar, goods, hgv, psv), or a
+#' cusstomized version dervied from those.
```

Comment:
This paragraph still has the same typos that were fixed by #248

Reply:
Fixed; thanks!